string serial_id
geometry_msgs/Pose2D goal_postion
---
geometry_msgs/Pose2D[] path